configs/baremetal.h

```diff
@@ -0,0 +1,113 @@
+/**
+ * \file baremetal.h
+ *
+ * \brief Test configuration for minimal baremetal Mbed TLS builds
+ *        based on the following primitives:
```

Comment:
Specifically, we want to support only 
`TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8` as specified in the LWM2M spec [here](http://openmobilealliance.org/RELEASE/LightweightM2M/V1_1-20180612-C/OMA-TS-LightweightM2M_Transport-V1_1-20180612-C.pdf). I think you can state that to make it clearer.

Reply:
I corrected the ciphersuite.